wondrous-app/components/OnboardingDao/ImportTasks/index.tsx

```diff
@@ -0,0 +1 @@
+export { default } from './component';
```

Comment:
same, if there's no other exports let's use the component itself in the index file